Product Designer/assets/css/styles.css

```diff
@@ -0,0 +1,203 @@
+/* UNIVERSAL STYLES */
+*{
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    color: white;
+    padding: 0;
+    box-sizing: border-box;
+    margin: 0px;
+}
+body{
+    background-color: black;
+}
+
+/* MAIL SECTION STYLES*/
+.mailSection{
+    width: 74%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: end;
+    font-size: 2.5em;
+    font-weight: 900;
+    height: 163px;
+    align-items: center;
+}
+.mailSection h6{
+    letter-spacing: 0.5px;
+    font-size: 0.6em;
+}
+
+/* NAME SECTION STYLES */
+.nameSection{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 485px;
+    width: 74%;
+    margin: 0 auto;
+}
+.name{
+    font-size: 150px;
+    margin: 0;
+}
+.name span{
+    font-weight: 300;
+    letter-spacing: 3px;
+}
+.role span{
+    color: #4b90e2;
+}
+.role{
+    font-size: 47px;
+}
+.mobileEmail{
+    display: none;
```

Comment:
if it is display none, why do we need this class ?

Reply:
Basically i have used 2 headers .... for the email id text 
one for laptop view port 
    - in laptop view port i don't want to display the the mobile email
and another one for the mobile view port 
    - in mobile view port i want to display this text 
    
This is why i have used this class